userspace/libsinsp/event.cpp

```diff
@@ -2198,10 +2207,19 @@ const char* sinsp_evt::get_param_as_str(uint32_t id, OUT const char** resolved_s
 			break;
 		}
 	case PT_ABSTIME:
-		//
-		// XXX not implemented yet
-		//
-		ASSERT(false);
+		{
+			ASSERT(payload_len == sizeof(uint64_t));
+			uint64_t val = *(uint64_t *)payload;
+			time_t sec = val / 1000000000ULL;
+			unsigned long nsec = val % 1000000000ULL;
+			struct tm tm;
+			localtime_r(&sec, &tm);
+			strftime(&m_paramstr_storage[0],
+				m_paramstr_storage.size(),
+				"%Y-%m-%d %H:%M:%S.XXXXXXXXX %z", &tm);
+			snprintf(&m_paramstr_storage[20], 9, "%09ld", nsec);
```

Comment:
minor: I haven't look much at the rest of this file, but would it makes sense to extract a helper being used in both places?

Reply:
hmm. maybe, but IIRC we have a ton of duplication between these two methods right now, so maybe we could take a more general approach and extract as much as we can in a separate PR?

(IMHO this isn't the important part of the PR, just something I ran into while trying to print the events with sysdig)

Reply:
BTW I just submitted the fix as a separate PR: https://github.com/falcosecurity/libs/pull/789 so it will get rebased out of this PR when that gets merged

Reply:
Thanks!